.github/workflows/license-lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: license-lint
+on: [push, pull_request]
+
+jobs:
+  license-lint:
+    name: "license-lint"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    permissions:
+      contents: read
+    steps:
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: license-lint
+        run: |
+          export GOPATH="$HOME/go"
+          PATH="$GOPATH/bin:$PATH"
```

Comment:
`setup-go` doesn't do these automatically?

Reply:
in my experience, GOPATH is not bound, unless set for this step. So we need to export these.